runmariadb.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/bash
+PATH=$PATH:/usr/sbin/
+DATADIR="/var/lib/mysql"
+
+crudini --set /etc/my.conf mysqld max_connects 64
+crudini --set /etc/my.conf mysqld max_heap_table_size 1M
+crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
+crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
+
+mysql_install_db --datadir="$DATADIR"
+
+chown -R mysql /var/log/mariadb
+chown -R mysql "$DATADIR"
+
+cat > /tmp/configure-mysql.sql <<-EOSQL
+    DELETE FROM mysql.user ;
+    CREATE USER 'ironic'@'localhost' identified by '${MARIADB_PASSWORD}' ;
```

Comment:
MARIADB_PASSWORD is undefined in this script.

Reply:
true, missed this as using it in conjunction with https://github.com/openshift-metalkube/dev-scripts/pull/241
I will fix it